cdap-cli/src/main/java/co/cask/cdap/cli/command/CreateStreamConversionAdapterCommand.java

```diff
@@ -130,15 +130,15 @@ public String getDescription() {
     return new StringBuilder()
       .append("Creates a stream conversion ")
       .append(ElementType.ADAPTER.getPrettyName())
-      .append(" that periodically reads from a stream and writes to a time partitioned fileset.")
+      .append(" that periodically reads from a stream and writes to a time partitioned fileset. ")
       .append(ArgumentName.FREQUENCY)
-      .append("is a number followed by a 'm', 'h', or 'd' for minute, hour, or day.")
+      .append("is a number followed by a 'm', 'h', or 'd' for minute, hour, or day. ")
```

Comment:
this is just how the StringBuilder (not our class) works. Append doesn't have to be a sentence so it can't auto-add spaces.